### DISCUSSION TITLE
51 days without Github Support replying

### DISCUSSION THREAD

**Discussion by DTtempo** [community/community#122123] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Yes, its been 51 days since I opened my support ticket and I have not received any answers or replies.

I work as a professional developer and during one of my days working, doing what I have been doing every day, my account suddenly got suspended without apparent reason as my actions did not really differ from any other day.

I opened up a support ticket to try and understand why my account was suspended and what I possibly need to do to reenable my account as I need it for my work. That was **51 days ago** and still I have not received any type of answer from the Github Support team.

Finally I asked if my employer could make a support ticket (as it concerns them as well) since they are on the PRO plan, which according to Github should have a response time of [24 hours](https://docs.github.com/en/enterprise-server@3.12/support/learning-about-github-support/about-github-support#support-in-english) as its covered by the standard support. That was 15 days ago and still no reply to neither my support ticket nor my employers.

I really do not think the forums should be used as a place for bumping forgotten support tickets, and I apologize to my fellow Github users and the support team, but I do not see any other means of contacting the Github Support team as the intended way of contacting them is evidently not working.

This throwaway account has been created for the sole reason to post this in the forums and I truly hope that it will not only bring some much needed attention to my support ticket which has the ID: **2670716** but to also highlight a problem that, judging from the forums, more users have been having as of late.

**Comment by asmawi117** [comment]
It's disappointing that despite being on the PRO plan, the response time hasn't met the promised 24-hour mark. I was about to upgrade to the Pro plan so my ticket could receive a faster response, but it seems it's useless anyway. I've had a ticket open since February 13, almost three months now.

**Comment by bryandbernhardt** [comment]
I understand that dealing with an account suspension can be quite frustrating, especially when it impacts your professional work. Here's a step-by-step guide to address the issue:

1. **Appeal and Reinstatement**: If you haven't already, fill out the GitHub Appeal and Reinstatement form (https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement). This is the official channel to appeal an enforcement action or seek reinstatement of your account.

2. **Follow-Up**: Since it's been 51 days without a response, it's reasonable to follow up on your ticket. You can do this by replying to the original email thread of your support ticket to bring it back to the attention of the support team.

3. **Employer's Support Ticket**: Given that your employer has a PRO plan, they should indeed have a faster response time. Encourage them to follow up on their ticket as well.

4. **Public Forum**: While it's not ideal to use public forums for support issues, in your case, it seems necessary to get the attention of GitHub Support. Make sure to include the ticket ID and explain the impact of the suspension on your work.

5. **Direct Contact**: If possible, try reaching out to GitHub via any direct contact methods they provide, such as social media channels, where they might respond more quickly to public queries.

6. **Patience and Persistence**: Continue to be patient but persistent. Support teams can be overwhelmed, and sometimes tickets require additional nudges to get noticed.

Remember, it's important to avoid sharing any sensitive personal information publicly while trying to resolve this issue. Stick to official channels and keep your communication professional and courteous.

If you've already taken some of these steps, consider them as reminders or additional options to explore. I hope your issue gets resolved soon.

(1) GitHub Appeal and Reinstatement - GitHub Docs. https://docs.github.com/site-policy/acceptable-use-policies/github-appeal-and-reinstatement.
(2) My account was suspended because I applied for a student ... - GitHub. https://github.com/orgs/community/discussions/63233.
(3) GitHub Account Got Suspended—Learn How To Appeal It Now - DoNotPay. https://donotpay.com/learn/github-account-suspended/.
(4) Suspending and unsuspending users - GitHub Docs. https://docs.github.com/enterprise/admin/guides/user-management/suspending-and-unsuspending-users.

**Reply by LiteBrite82** [reply]
Hi @bryandbernhardt , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
